examples/nextjs-tiptap-novel/src/components/editor/ui/menu.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import { Check, Menu as MenuIcon, Monitor, Moon, SunDim } from "lucide-react";
+import { useTheme } from "next-themes";
+import { Button } from "./button";
+import { Popover, PopoverContent, PopoverTrigger } from "./popover";
+
+// TODO implement multiple fonts editor
+// const fonts = [
+//   {
+//     font: "Default",
+//     icon: <FontDefault className="h-4 w-4" />,
+//   },
+//   {
+//     font: "Serif",
+//     icon: <FontSerif className="h-4 w-4" />,
+//   },
+//   {
+//     font: "Mono",
+//     icon: <FontMono className="h-4 w-4" />,
+//   },
+// ];
```

Comment:
Should we remove this?

Reply:
I've left a bunch of TODOs in because they're from the official Novel editor. I've done this to make it easier for people to diff and see what's changed. Let me know if you think this is a bad idea!